tools/schema/cardBaseSchema.json

```diff
@@ -54,18 +54,15 @@
             "maxLength": 20,
             "pattern": "^[a-z]+_[0-9a-z]+$"
           },
+          "direction": {
```

Comment:
To my understanding, we're not using a direction member, because links are recorded in the "from" card only, so the direction member should be removed from here and also from the "required" member.

Reply:
App is using 'direction' when it is handling link creation in UI.